common/crypto/testdata/private-key.ecdsa.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIMLuuigwK0YFIWUPnccCPXhYThK+Mk7vc2EfiMqRmV+JoAoGCCqGSM49
+AwEHoUQDQgAEX/+x/k2uzt87N/Ms1QTUzTMz6kuMuJNsr3VsPaQzhIyhdl5Z9Isq
+CpinTBnM0o2L7jRq70g9ANs6/P5dPlK7TA==
+-----END EC PRIVATE KEY-----
```

Comment:
We could rename the old keys to `private-key.rsa.pem` and `public-key.rsa.pem`.

Reply:
Good point! Will do!